.github/workflows/main.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         uses: actions/setup-dotnet@3447fd6a9f9e57506b15f895c5b76d3b197dc7c2 # 3.2.0
         with:
           dotnet-version: 8.0.x
```

Comment:
🌱 Since you're using `global.json` none of your usages of this Action need to specify a .NET version and it'll read that file on its own. Easier to maintain.